src/test/java/core/basesyntax/service/impl/TransactionStrategyImplTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitShopOperation;
+import core.basesyntax.service.interfaces.TransactionStrategy;
+import core.basesyntax.service.transaction.TransactionHandler;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class TransactionStrategyImplTest {
+    private TransactionStrategy transactionStrategy;
+
+    @BeforeEach
+    void setUp() {
+        transactionStrategy = new TransactionStrategyImpl();
+    }
+
+    @Test
+    void getReturnHandler_OK() {
+        TransactionHandler returnHandler =
+                TransactionStrategyImpl.handlers.get(FruitShopOperation.RETURN);
+        Assertions.assertEquals(returnHandler,
+                transactionStrategy.get(FruitShopOperation.RETURN));
+    }
+
+    @Test
+    void getSupplyHandler_OK() {
+        TransactionHandler supplyHandler =
+                TransactionStrategyImpl.handlers.get(FruitShopOperation.SUPPLY);
+        Assertions.assertEquals(supplyHandler,
```

Comment:
this assert is kind of useless for me, because it similar to: assertEquals(actual, actual) is it true always, better to make assertEquals(SomeHandles.class, actual.getClass)